ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh

```diff
@@ -75,6 +75,13 @@ if [ "${DISCONNECTED}" == "true" ] && [ -f "${SHARED_DIR}/install-config-mirror.
 fi
 file /tmp/openshift-baremetal-install
 
+
+if [ "${EXTERNAL_LB}" == "true" ] && [ -f "${SHARED_DIR}/external_vips.yaml" ]; then
+  VIPS="${SHARED_DIR}/external_vips.yaml"
+else
+  VIPS="${SHARED_DIR}/vips.yaml"
+fi
+
```

Comment:
This is going to be a potential issue for future implementations using other LBs (as we already have). We are trying to avoid adding new variables when they are not really needed. 

Can we rather limit the changes for this bugfix to the provisioning network pre and post steps?

As an example, we can just use the last octet of the first master's IPv4 and we should actually solve a TODO we had in the pre step's code too wrt ipv6.

Reply:
Done